### PR TITLE
Add python 3.8 testing

### LINKED ISSUE #154: Enable and fix integration tests against devel
##### SUMMARY
The integration tests against `devel` were disabled in https://github.com/ansible-collections/community.mysql/pull/153  presumably because of https://github.com/ansible-collections/overview/issues/45#issuecomment-825453639

### PR DESCRIPTION
**Pull request by Jorge-Rodriguez**
##### SUMMARY
Adds Pyton 3.8 integration testing. This supports testing of Ansible devel, for which python 3.8 is a requirement.
Fixes #154 

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

mysql collection

### CODE REVIEW

File: .github/workflows/ansible-test-plugins.yml [hunk 1]
```diff
@@ -61,13 +61,17 @@ jobs:
...
         connector:
           - pymysql==0.7.10
           - pymysql==0.9.3
           - mysqlclient==2.0.1
+        include:
+          - ansible: devel
+            python: 3.8
         exclude:
           - mysql: 8.0.22
             connector: pymysql==0.7.10
```
Comment (unnecessary-username) [inline .github/workflows/ansible-test-plugins.yml:77]: I think this part could be like:
```
          - devel
        python:
          - 3.6
          - 3.8
        connector:
          - pymysql==0.7.10
          - pymysql==0.9.3
          - mysqlclient==2.0.1
        exclude:
          - mysql: 8.0.22
            connector: pymysql==0.7.10
          - python: 3.8
            ansible: stable-2.9
          - python: 3.8
            ansible: stable-2.10
          - python: 3.8
            ansible: stable-2.11
          - python: 3.6
            ansible: devel
```

**Comment by Andersson007** [comment]
@Jorge-Rodriguez thanks for the PR!
It doesn't show test logs now because "they expired". I'll close and reopen it to relaunch the tests. interesting what's there.

**Comment by Andersson007** [comment]
In case it'll get stale again quickly, i put it here
```
TASK [setup_mysql : setup_mysql | setvars | get mysql major version] ***********
task path: /root/ansible_collections/community/mysql/tests/output/.tmp/integration/test_mysql_variables-hstbvwg5-ÅÑŚÌβŁÈ/tests/integration/targets/setup_mysql/tasks/setvars.yml:6
fatal: [testhost]: FAILED! => {
    "msg": "The task includes an option with an undefined variable. The error was: list object has no element 1\n\nThe error appears to be in '/root/ansible_collections/community/mysql/tests/output/.tmp/integration/test_mysql_variables-hstbvwg5-ÅÑŚÌβŁÈ/tests/integration/targets/setup_mysql/tasks/setvars.yml': line 6, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: \"{{ role_name }} | setvars | get mysql major version\"\n  ^ here\nWe could be wrong, but this one looks like it might be an issue with\nmissing quotes. Always quote template expression brackets when they\nstart a value. For instance:\n\n    with_items:\n      - {{ foo }}\n\nShould be written as:\n\n    with_items:\n      - \"{{ foo }}\"\n"
}
```

**Comment by unnecessary-username** [comment]
> In case it'll get stale again quickly, i put it here
> 
> ```
> TASK [setup_mysql : setup_mysql | setvars | get mysql major version] ***********
> task path: /root/ansible_collections/community/mysql/tests/output/.tmp/integration/test_mysql_variables-hstbvwg5-ÅÑŚÌβŁÈ/tests/integration/targets/setup_mysql/tasks/setvars.yml:6
> fatal: [testhost]: FAILED! => {
>     "msg": "The task includes an option with an undefined variable. The error was: list object has no element 1\n\nThe error appears to be in '/root/ansible_collections/community/mysql/tests/output/.tmp/integration/test_mysql_variables-hstbvwg5-ÅÑŚÌβŁÈ/tests/integration/targets/setup_mysql/tasks/setvars.yml': line 6, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: \"{{ role_name }} | setvars | get mysql major version\"\n  ^ here\nWe could be wrong, but this one looks like it might be an issue with\nmissing quotes. Always quote template expression brackets when they\nstart a value. For instance:\n\n    with_items:\n      - {{ foo }}\n\nShould be written as:\n\n    with_items:\n      - \"{{ foo }}\"\n"
> }
> ```

The python 3.8 testing job cannot get MySQL version and connector so it failed. I guess the include option need to state every argument explicitly, so I would suggest to use the exclude option to achieve this combination.

**Comment by Andersson007** [comment]
I've created a PR https://github.com/ansible-collections/community.mysql/pull/226, please take a look, folks

**Pull request closed without merging**